Gemfile

```diff
@@ -1,12 +1,9 @@
 source "http://rubygems.org"
-# Add dependencies required to use your gem here.
-# Example:
 
-gem 'activesupport'
+gem 'activesupport', '~> 5.1', '>= 5.1.4'
```

Comment:
Probably not worth it, but if other people are using this gem, requiring rails 5 may break things for them and we can probably live with an older version of `activesupport`...

Reply:
True -- I went with this version because it's what we had in the Gemfile.lock and so I'm sure it works. Since there are 0 stars on this repo, and I think we know basically all the people following it, I think we're probably safe to require rails 5?